### DISCUSSION TITLE
Shifting tool, more explanations please

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#649] [Q&A] [answered]
Hello,

Could you explain more the shifting tool in advanced settings? Is it enabled by default? If so how can I disable it? 

Can I use it to reduce background noises?

Thank you.

**Comment by ehpersonal38** [comment, marked as answer]
It's simply a feature that allows you to pitch down a recording by clicking the "freq shift" button on the Recordings page. It was originally implemented by someone with hearing difficulties such that he couldn't hear high frequencies, so pitching down the bird sounds allowed him to hear them.

It's not enabled by default, it only affects recordings that you manually press the button on.

Reducing background noise is not something it can be used for. That's a much more complicated feature that I'm not really sure how to implement.

**Reply by blogdemoi** [reply]
> It's not enabled by default, it only affects recordings that you manually press the button on.

Thank you. So how can I enable it if I wish? I don't see any buttons...

**Reply by ehpersonal38** [reply]
It's just to the right of the delete button and the purge exclude button (the one that looks like a lock), on any individual recording. To access an individual recording, click the "open in New tab" link in the top right of a recording on Overview or Today's Detections, or simply navigate to it directly from Recordings -> (pick one)

**Reply by blogdemoi** [reply]
Thank you.
